circbuf_test.go

```diff
@@ -196,3 +196,46 @@ func TestBuffer_Reset(t *testing.T) {
 		t.Fatalf("bad: %v", string(buf.Bytes()))
 	}
 }
+
+func TestBuffer_WriteByte(t *testing.T) {
+	inp := []byte("hello world")
+
+	buf, err := NewBuffer(3)
+	if err != nil {
+		t.Fatalf("err: %v", err)
+	}
+
+	for _, b := range inp {
+		buf.WriteByte(b)
+	}
+
+	expect := []byte("rld")
+	actual := buf.Bytes()
+	if !bytes.Equal(actual, expect) {
+		t.Fatalf("bad: %v", actual)
+	}
+}
+
+func TestBuffer_ReadByte(t *testing.T) {
+	inp := []byte("hello world")
+
+	buf, err := NewBuffer(int64(len(inp)))
+	if err != nil {
+		t.Fatalf("err: %v", err)
+	}
+
+	if _, err := buf.Write([]byte("hell")); err != nil {
```

Comment:
Is it possible to be more descriptive in the errors prints?

Reply:
@zozo123 not sure what you mean -- which error prints?

Reply:
see above. ignore this comment